README.md

```diff
@@ -2,62 +2,160 @@
+"more powerful."
+
+Developers love the command line and, as a result, write most developer tools
+for other users of the command line. As a result, tools you will need to be a
+developer will require you to use them through the command-line interface.
+
+Lots of people worry that using the CLI will get them in trouble - like they'll
+break their computer. We'd encourage you to think about the CLI like a good
+sharp kitchen knife: if you pay attention when you use it, you're going to have
+a valuable ally on your side. Take your time and if you have a question, ask.
+
+## Identify Differences Between Command-Line Interface, Terminal Wmulator, and Shell
+
+When you want to use the CLI, you launch a program called a "terminal" program.
+That's short for "terminal emulator." A long time ago, people only had keyboards
+and monitors (no mouse or graphic interface!) that were tied to a computer that
+they all shared. This monitor + keyboard device was called a "terminal." It looked
+like this:
+
+< pic of terminal >
+
+< graphic of terminal cabled to a shared computer >
+
+Nowadays, the "terminal" is "emulated" in software. It's "virtual." You launch
+the "emulator" by opening a program called `Terminal` on OSX or something
+similar on Linux or Windows. Instead of being connected to a remote computer by
+a cable, your "terminal emulator" talks to the computer you're actually typing
+on.
+
+< graphic of terminal cabled to itself >
+
+When you launch the "terminal emulator" program, it will immediately start a
+program called a _shell_ program. The _shell_ program is what actually prompts
+you for input and returns the output. The shell most computers default to
+using is known as `bash`.
+
+To help keep these terms straight, here's a table
+
+< insert table derived from style guide >
+
+## Demonstrate How to Navigate With `bash`
+
+To sum up: `bash` is a text-based interpreter that provides a _command-line
+interface_ for controlling your computer (or operating system).
+
+> **ASIDE**: Bash is actually an acronym which stands for **B**ourne-**A**gain
+> **SH**ell. As the word "again" suggests, there are _other_ shells, some of
+> which came before `bash`. There are also shells that have come along _since_
+> `bash`. Nevertheless most programmers use `bash` or something very similar.
+
+### Identify My Logged-In Username
+
+Let's start simply. Let's ask the computer who I am logged in as:
+
+```bash
+$ whoami
+```
+
+ < probably need screen shot>
+
+The `whoami` command lets you see which user account you're logged in to from a
+terminal window. This might seem obvious, especially if you're logged in on your
+personal computer, but it's not always clear which account you're you're working
+in the command line.
+
+In a strange circumstance where `whoami` isn't installed, there is another command
+you can use that can tell you your current username. Try:
+
+```bash
+$ id -un
+```
 
 ### Identifying the "Present Working Directory" With `pwd`
```

Comment:
Add:

Introduce idea of file-system. We use this word below and assume they know what it is.

Present working Direcotry and pwd command are murky unless you est. the context that we're going to do navigation.